### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by lugard1**
# Description

- [ ] Update index.html, index.js, style.css, and other files.
- [ ] Update web-pack and linters package files
- [ ] Fixed linter errors
- [ ] Update Readme file.

### CODE REVIEW

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @lugard1  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good project setup :heavy_check_mark: 
- No linters errors :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please make sure in your next project to have a descriptive branch name instead of **feature1**  . You can put something more descriptive :+1: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  border: none;
+  background-color: blueviolet;
+  padding: 20px 0;
+  margin: auto;
+  font-size: 16px;
+  cursor: pointer;
+  text-align: center;
+  color: #fff;
+  border-radius: 0 0 8px 8px;
+}
```
[the remark this answers is not in the document]
Comment (lugard1) [reply src/style.css]: made my design a little bit similar to the design captured in the video

File: src/modules/add.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    taskArray.forEach((task) => {
+      taskArrayDOM += `
+          <li><span><input class="checkbox" type="checkbox" name="taskArray";
+          isChecked${task.completed};><input class="input" type="text" name="taskArrayi" value="${task.description}"></span>
+          <i class="fa-solid fa-ellipsis-vertical"></i></li><br>`;
+    });
+    isChecked();
+    return taskArrayDOM;
+  }
+}
```
[the remark this answers is not in the document]
Comment (lugard1) [reply src/modules/add.js]: The list is made now to appear in order of the index values for each task.

**Review by ichala — APPROVED** [review]
Hi @lugard1   ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**